### PR TITLE
Specify correct pixel scale in window dimensions to support multi-res assets in high-res display.

### PR DESCRIPTION
**Pull request by suyuan-chang**
So that image asset loader can load best fit image asset for high resolution display.

### CODE REVIEW

File: ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using ReactNative.Modules.Core;
 using System.Collections.Generic;
 using System.Windows;
+using System;
```
Comment (matthargett) [inline ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs]: Sort usings. System goes before System.Windows.

File: ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs [hunk 2]
```diff
@@ -84,6 +85,12 @@ private void OnBoundsChanged(object sender, SizeChangedEventArgs args)
             var content = (FrameworkElement)_window.Content;
             double scale = 1.0;
 
+            IntPtr hwnd = new System.Windows.Interop.WindowInteropHelper(_window).Handle;
```
Comment (matthargett) [inline ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs]: Use var keyword

File: ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs [hunk 3]
```diff
@@ -84,6 +85,12 @@ private void OnBoundsChanged(object sender, SizeChangedEventArgs args)
...
+            using (System.Drawing.Graphics g = System.Drawing.Graphics.FromHwnd(hwnd))
```
Comment (matthargett) [inline ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs]: Also use var keyword here

File: ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs (hunk 2 above)
Comment (suyuan-chang) [inline ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs]: May I know why here prefer `var` keyword instead of explicit type name?

**Comment by suyuan-chang** [comment]
Code update. Please review it again. Thanks.

File: ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs [hunk 4]
```diff
@@ -84,6 +84,12 @@ private void OnBoundsChanged(object sender, SizeChangedEventArgs args)
             var content = (FrameworkElement)_window.Content;
             double scale = 1.0;
 
+            var hwnd = new System.Windows.Interop.WindowInteropHelper(_window).Handle;
+            using (var g = System.Drawing.Graphics.FromHwnd(hwnd))
+            {
+                scale = g.DpiX / 96;
```
Comment (matthargett) [inline ReactWindows/ReactNative.Net46/Modules/DeviceInfo/DeviceInfoModule.cs]: is the 96 query-able from the OS? some people configure this differently

**Comment by rozele** [comment]
Closing, but we should still review what the correct DPI scaling behavior should be on WPF.

**Pull request closed without merging**